src/main/java/gov/nih/nci/evs/api/model/Relationship.java

```diff
@@ -84,7 +80,6 @@ public void populateFrom(final Relationship other) {
    *
    * @return the code
    */
-  @Schema(hidden = true)
```

Comment:
Everywhere you're removing a @Schema(hidden = true) you actually ahve to instead replace it with a @Schema that has a description (borrow one from one of the other fields to see how we do it)